README.md

```diff
@@ -1 +1,38 @@
-# create-and-manage-post
\ No newline at end of file
+# create-and-manage-post
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+## Running the JSON(dummy) Server
```

Comment:
This isn't what I referred to as the `JSON(dummy) Server`. I was referring to the JSON file, `db.json` that you use as a temporal backend.
```
json-server --watch src/db/db.json
```
Or how do you run your backend?